src/transformers/models/lxmert/modeling_lxmert.py

```diff
@@ -739,7 +739,7 @@ def __init__(self, config):
             visual_losses["obj"] = {"shape": (-1,), "num": config.num_object_labels}
         if config.visual_attr_loss:
             visual_losses["attr"] = {"shape": (-1,), "num": config.num_attr_labels}
-        if config.visual_obj_loss:
+        if config.visual_feat_loss:
```

Comment:
This was like this since the model was added. But I feel it's very likely a mistake. Otherwise, `config.visual_feat_loss` is currently not used at all.